pymongo_auth_aws/auth.py

```diff
@@ -185,6 +226,17 @@ def _aws_auth_header(credentials, server_nonce, sts_host):
     return final
 
 
+def _handle_credentials(func):
+    def inner(self, *args, **kwargs):
```

Comment:
It's good practice to use `@functools.wraps(func)` here so that the original function name and docstring is retained after the decorator is applied: https://docs.python.org/3/library/functools.html#functools.wraps

Reply:
Done